include/game/bosses/common.h

```diff
@@ -7,6 +7,18 @@
 extern struct Task *gActiveBossTask;
 typedef void (*TranslateBossFunction)(s32, s32);
 
+typedef enum {
```

Comment:
Interesting to use an enum here instead of defines? This is the first time we've done this?

Reply:
I don't know, it's a little more ordered, I think?
Though I agree that consistnecy is key! We could think about it when we're at / closer to 100%.